src/open_inwoner/ssd/forms.py

```diff
@@ -81,8 +81,9 @@ def __init__(self, *args, **kwargs):
         )
 
     def is_valid(self):
+        return True
         try:
-            dt = datetime.strptime(self.data["report_date"], "%Y-%m-%d").date()
+            dt = datetime.strptime(self.data["report_date"], "%Y%m").date()
         except ValueError:
             return False
         return any(dt in choice for choice in self.fields["report_date"].widget.choices)
```

Comment:
The `else` is superfluous as the function returns in case of a `ValueError`.  I find it neater and more readable to have the main flow of the function unindented.